modin/pandas/test/test_series.py

```diff
@@ -2192,7 +2217,14 @@ def test_max(data, skipna):
 )
 def test_mean(data, skipna):
     modin_series, pandas_series = create_test_series(data)
-    df_equals(modin_series.mean(skipna=skipna), pandas_series.mean(skipna=skipna))
+    try:
+        pandas_result = pandas_series.mean(skipna=skipna)
+    except Exception as e:
+        with pytest.raises(type(e)):
+            modin_series.mean(skipna=skipna)
+    else:
+        modin_result = modin_series.mean(skipna=skipna)
+        df_equals(modin_result, pandas_result)
```

Comment:
```suggestion
    eval_general(*create_test_series(data), lambda df: df.mean(skipna=skipna))
```
The same comments as for `test_all` func

Reply:
Fixed.